conversational-memory/src/main/java/org/opensearch/ml/memory/action/conversation/CreateConversationResponse.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Aryn
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.opensearch.ml.memory.action.conversation;
+
+import java.io.IOException;
+
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.ToXContentObject;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.ml.common.conversation.ActionConstants;
+
+import lombok.AllArgsConstructor;
+
+/**
+ * Action Response for CreateConversation
+ */
+@AllArgsConstructor
+public class CreateConversationResponse extends ActionResponse implements ToXContentObject {
+
+    String conversationId;
+
+    /**
+     * Convtructor
```

Comment:
minor, but I pointed out couple times

Reply:
fixed this in the next PR (feature flag)